crates/swc_css_parser/src/lexer/mod.rs

```diff
@@ -267,31 +288,25 @@ where
             // U+003A COLON (:)
             // Return a <colon-token>.
             Some(c) if c == ':' => {
-                self.input.bump();
-
                 return Ok(tok!(":"));
             }
             // U+003B SEMICOLON (;)
             // Return a <semicolon-token>.
             Some(c) if c == ';' => {
-                self.input.bump();
-
                 return Ok(tok!(";"));
             }
             // U+003C LESS-THAN SIGN (<)
             Some(c) if c == '<' => {
-                self.input.bump();
-
                 // If the next 3 input code points are U+0021 EXCLAMATION MARK U+002D
                 // HYPHEN-MINUS U+002D HYPHEN-MINUS (!--), consume them and return a
                 // <CDO-token>.
-                if self.input.is_byte(b'!')
```

Comment:
Actually `is_byte` is an optimization and it's much faster than comparing `char`.

Reply:
It was one place where we used `is_byte`, but if it is faster we can rewrite `self.cur()` and `self.next()`/etc on bytes, I tried it right now, I don't see any perf, maybe I missed something?